src/04-arrays.js

```diff
@@ -135,7 +157,12 @@ function removeElementFromFrontOfArray(array) {}
  *  getMiddleElement([ 10, null, "30" ]);
  *  //> null
  */
-function getMiddleElement(array) {}
+function getMiddleElement(array) {
+  let mid = [array[array.length / 2 - 1]];
```

Comment:
Lets be careful with our ` [] ` here.  array does not need a bracket around it.  Also lets be careful where we are indexing our ` array `.  What happens with this code if the  ` array[array.length/2] ` turns out to be 1.5?  Can we index an array like this?